tests/model_hub_tests/tensorflow/utils.py

```diff
@@ -139,3 +142,26 @@ def get_output_signature(graph: tf_v1.Graph):
             if op.type not in unlikely_output_types:
                 outputs.append(op.name + ':0')
     return outputs
+
+
+def try_get_signature(model):
+    if type(model) is tf_v1.Graph:
+        return get_input_signature_from_tf_graph(model)
+    elif hasattr(model, "input_signature") and model.input_signature is not None:
+        return model.input_signature
+    elif hasattr(model, "structured_input_signature") and model.input_signature is not None:
+        assert len(model.structured_input_signature) > 1, "incorrect model or test issue"
+        return model.structured_input_signature[1].items()
+    else:
+        return None
+
+
+def get_input_signature(model):
+    signature = try_get_signature(model)
+    if signature is None:
+        if hasattr(model, "model"):
+            signature = try_get_signature(model.model)
+    if signature is None:
+        raise Exception("Could not extract signature from original model.")
+    return signature
+
```

Comment:
let us add models to precomit that reproduce this issue

Reply:
Done.